configs/configgen.sh

```diff
@@ -3,14 +3,19 @@
 set -e
 
 SCRIPT_DIR=`dirname $0`
-BUILD_DIR=$2/configgen
+BUILD_DIR=/tmp/configgen
+OUT_DIR=$1
+shift
+
 if [ ! -d $BUILD_DIR/venv ]; then
   virtualenv $BUILD_DIR/venv
   $BUILD_DIR/venv/bin/pip install -r $SCRIPT_DIR/requirements.txt
 fi
 
-mkdir -p $1
-$BUILD_DIR/venv/bin/python $SCRIPT_DIR/configgen.py $1
-cp -f $SCRIPT_DIR/google_com_proxy.json $1
-cp -f $SCRIPT_DIR/../examples/front-proxy/{front-envoy,service-envoy}.json $1
-cp -f $SCRIPT_DIR/../examples/grpc-bridge/config/*.json $1
+mkdir -p $OUT_DIR
```

Comment:
Nit: generally prefer quoting of variables due to bash wsp issues, e.g. "${OUT_DIR}".